src/v2/guide/render-function.md

```diff
@@ -436,60 +436,60 @@ new Vue({
 
 ``` js
 Vue.component('my-component', {
   functional: true,
-  // To compensate for the lack of an instance,
-  // we are now provided a 2nd context argument.
+  // Pour compenser le manque d'instance,
+  // nous pouvons maintenant fournir en second argument un contexte.
   render: function (createElement, context) {
     // ...
   },
-  // Props are optional
+  // Les props sont optionnelles
   props: {
     // ...
   }
 })
 ```
 
-> Note: in versions <=2.3.0, the `props` option is required if you wish to accept props in a functional component. In 2.3.0+ you can omit the `props` option and all attributes found on the component node will be implicitly extracted as props.
+> Note : dans les versions <= 2.3.0, l'option `props` est requise si vous souhaitez accepter des props dans un composant fonctionnel. Dans les versions 2.3.0+ vous pouvez omettre l'option `props` et tous les attributs trouvés dans le nœud composant seront implicitement extrait comme des props.
 
-Everything the component needs is passed through `context`, which is an object containing:
+Tout ce dont le composant a besoin est passé dans l'objet `context`, qui est un objet contenant :
 
-- `props`: An object of the provided props
-- `children`: An array of the VNode children
-- `slots`: A function returning a slots object
-- `data`: The entire data object passed to the component
-- `parent`: A reference to the parent component
-- `listeners`: (2.3.0+) An object containing parent-registered event listeners. This is simply an alias to `data.on`
-- `injections`: (2.3.0+) if using the [`inject`](../api/#provide-inject) option, this will contain resolved injections.
+- `props`: un objet avec les props fournies,
+- `children`: un tableau de VNode enfant,
+- `slots`: une fonction retournant un objet de slots,
+- `data`: l'objet de données (`data`) complet passé au composant,
+- `parent`: une référence au composant parent,
+- `listeners`: (2.3.0+) un objet contenant les écouteurs d'évènement enregistré du parent. C'est un simple alias de `data.on`,
+- `injections`: (2.3.0+) si vous utilisez l'option [`inject`](../api/#provide-inject), cela va contenir les injections résolues.
 
-After adding `functional: true`, updating the render function of our anchored heading component would simply require adding the `context` argument, updating `this.$slots.default` to `context.children`, then updating `this.level` to `context.props.level`.
+Après avoir ajouté `functional: true`, mettre a jour la fonction de rendu de votre composant de titres avec ancres va seulement demander l'utilisation de l'argument `context`, la mise à jour de `this.$slots.default` pour `context.children`, puis la mise à jour de `this.level` pour `context.props.level`.
```

Comment:
mettre à jour

de notre composant

va simplement nécessiter d'ajouter l'argument `context`, en remplaçant `this.$slots.default` par `context.children`, puis  `this.level` par `context.props.level`.